Model/Config/Source/Available/Available.php

```diff
@@ -178,91 +184,99 @@ protected function _isAvailable()
         return false;
     }
 
-    public function setDefaults($Paymentmethods)
+    public function setDefaults()
     {
-
-        $Magento_paymentmethod_codes = array(
-            "0"    => "paylink",
-            "739"  => "afterpay",
-            "2080" => "alipay",
-            "1705" => "amex",
-            "2277" => "applepay",
-            "1672" => "billink",
-            "1744" => "capayable",
-            "1813" => "capayable_gespreid",
-            "1945" => "cartasi",
-            "710"  => "cartebleue",
-            "1981" => "cashly",
-            "139"  => "clickandbuy",
-            "2107" => "creditclick",
-            "1939" => "dankort",
-            "2062" => "eps",
-            "815"  => "fashioncheque",
-            "1699" => "fashiongiftcard",
-            "1702" => "focum",
-            "812"  => "gezondheidsbon",
-            "694"  => "giropay",
-            "1657" => "givacard",
-            "2283" => "huisentuincadeau",
-            "10"   => "ideal",
-            "1729" => "instore",
-            "1717" => "klarna",
-            "2265" => "klarnakp",
-            "712"  => "maestro",
-            "436"  => "mistercash",
-            "2271" => "multibanco",
-            "1588" => "mybank",
-            "136"  => "overboeking",
-            "2379" => "payconiq",
-            "138"  => "paypal",
-            "553"  => "paysafecard",
-            "816"  => "podiumcadeaukaart",
-            "707"  => "postepay",
-            "2151" => "przelewy24",
-            "559"  => "sofortbanking",
-            "1987" => "spraypay",
-            "1600" => "telefonischbetalen",
-            "2104" => "tikkie",
-            "706"  => "visamastercard",
-            "1714" => "vvvgiftcard",
-            "811"  => "webshopgiftcard",
-            "1978" => "wechatpay",
-            "1666" => "wijncadeau",
-            "1877" => "yehhpay",
-            "1645" => "yourgift",
-        );
-
-        $changed = false;
-
-        foreach ($Paymentmethods as $key => $method) {
-
-            if (isset($Magento_paymentmethod_codes[$method['id']])) {
-                $name = $Magento_paymentmethod_codes[$method['id']];
-
-                if (strlen($this->store->getConfig('payment/paynl_payment_' . $name . '/min_order_total')) == 0 && isset($method['min_amount'])) {
-                    $this->configWriter->save('payment/paynl_payment_' . $name . '/min_order_total', $method['min_amount']);
-                    if (strlen($method['min_amount']) > 0) {$changed = true;}
+        $configured = $this->configureSDK();
+        if ($configured) {
+            $Paymentmethods = Paymentmethods::getList();
+            $Magento_paymentmethod_codes = array(
```

Comment:
Deze array met profiles kan er volgens mij uit.

Ik mis nog steeds de afbeeldingen?

Reply:
Zoals ik zei, afbeeldingen worden niet gebruikt voor ideal in magento.

Die array kan er helaas niet uit want deze matched de naam die de payment optie gebruikt in magento met de van de getlist functie. Heb jij misschien hier een betere oplossing voor?